frontend/public/components/graphs/gauge.tsx

```diff
@@ -10,104 +10,106 @@ import { getInstantVectorStats } from './utils';
 const DEFAULT_THRESHOLDS = [{ value: 67 }, { value: 92 }];
 
 export const GaugeChart: React.FC<GaugeChartProps> = ({
-  query = '',
-  thresholds = DEFAULT_THRESHOLDS,
+  data,
+  error = false,
   invert = false,
-  usedLabel = 'used',
   label,
+  loading,
+  query = '',
   remainderLabel = 'available',
-  title,
   theme,
-  data,
+  thresholds = DEFAULT_THRESHOLDS,
+  title,
+  usedLabel = 'used',
+
+  // Don't sort, Uses previously declared props
   secondaryTitle = usedLabel,
-  error = false,
 }) => {
   const [ref, width] = useRefWidth();
   const labels = (d) => d.x ? `${d.x} ${usedLabel}` : `${d.y} ${remainderLabel}`;
-  return (
-    <PrometheusGraph className="graph-wrapper--title-center" title={title}>
-      <div ref={ref} className="graph-wrapper--gauge">
-        <PrometheusGraphLink query={query}>
-          <ChartDonutThreshold
-            data={thresholds}
-            height={width} // Changes the scale of the graph, not actual width and height
-            y="value"
-            width={width}
-          >
-            <ChartDonutUtilization
-              labels={labels}
-              data={error ? {y: 0}: data}
-              invert={invert}
-              subTitle={error ? null : secondaryTitle}
-              themeColor={ChartThemeColor.green}
-              thresholds={thresholds}
-              title={error ? 'No Data' : label}
-              theme={theme}
-            />
-          </ChartDonutThreshold>
-        </PrometheusGraphLink>
-      </div>
-    </PrometheusGraph>
-  );
+  return <PrometheusGraph className="graph-wrapper--title-center graph-wrapper--gauge" ref={ref} title={title}>
+    <PrometheusGraphLink query={query}>
+      <ChartDonutThreshold
+        data={thresholds}
+        height={width} // Changes the scale of the graph, not actual width and height
+        y="value"
+        width={width}
+      >
+        <ChartDonutUtilization
+          labels={labels}
+          data={error ? { y: 0 } : data}
+          invert={invert}
+          subTitle={error || loading ? null : secondaryTitle}
+          themeColor={ChartThemeColor.green}
+          thresholds={thresholds}
+          title={loading ? 'Loading' : error ? 'No Data' : label}
```

Comment:
Nit: Add either parentheses or whitespace to make this easier to read. Or use a variable.